wolfhsm/wh_comm.h

```diff
@@ -32,7 +32,8 @@
 enum {
     WH_COMM_HEADER_LEN = 8,    /* whCommHeader */
     WH_COMM_DATA_LEN = 1280,
-    WH_COMM_MTU = (WH_COMM_HEADER_LEN + WH_COMM_DATA_LEN)
+    WH_COMM_MTU = (WH_COMM_HEADER_LEN + WH_COMM_DATA_LEN),
+    WH_COMM_MTU_8 = (WH_COMM_MTU + 7) / 8,
```

Comment:
I think we should call it `WH_COMM_MTU_BITS`, wolfSSL defines things as bits when referring to sizes in bits: `#define SP_INT_BITS 4096`

Reply:
Hah!  This is actually the rounded-up number of 8-byte uint64_t's needed to hold an MTU packet.  How about I change this to WH_COMM_MTU_64?

Reply:
ah gotcha yeah thats much less confusing, I'd also add a comment

Reply:
went with WH_COMM_MTU_U64_COUNT with a comment.  Fixed in next push.